ui/packages/catalog-ui-search/src/main/webapp/react-component/location/point-radius.js

```diff
@@ -237,9 +237,16 @@ const PointRadiusUsngMgrs = props => {
           type="number"
           value={String(radius)}
           onChange={value => {
-            setRadiusError(validateGeo('radius', value))
             setState({ ['radius']: value })
           }}
+          onBlur={e =>
```

Comment:
huh, it looks like we set this error in the `onBlur` for all the other Radius components, but did it here for MGRS in the `onChange`. I don't think we had any reason to do it this way IIRC, @hayleynorton ?

Reply:
Not that I recall. Could just be we switched some things around but missed this one